manifests/bucketeer/charts/event-persister-evaluation-events-kafka/values.yaml

```diff
@@ -32,6 +32,8 @@ env:
   redis:
     serverName:
     addr:
+    poolMaxIdle: 50
+    poolMaxActive: 200
```

Comment:
Should we change these params?

Reply:
Honestly, I'm not sure how to decide it.
We can experiment with it in the dev, load, or production environment 👍

Reply:
Let's keep both at 50.